tests/Unit/Test/Normalizer/TestNormalizerTest.php

```diff
@@ -95,21 +97,107 @@ public function testNormalize(): void
+                                'qti-type' => 'timeLimits',
+                                'maxTime' => 60,
+                            ],
+                        ],
+                    ],
+                ],
+                [
+                    'qti-type' => 'testPart',
+                    'identifier' => 'cluster-id-stage1-Unit11',
+                    'navigationMode' => 0,
+                    'assessmentSections' => [
+                        [
+                            'qti-type' => 'assessmentSection',
+                            'identifier' => 'assessmentSection-1',
+                            'categories' => [],
+                            'sectionParts' => [
+                                [
+                                    'href' => 'https://itemUri0',
+                                    'qti-type' => 'assessmentItemRef',
+                                    'categories' => ['cluster-id-stage1', 'x-tao-option-reviewScreen'],
+                                    'identifier' => 'item-0',
+                                    'timeLimits' => [
+                                        'qti-type' => 'timeLimits',
+                                        'maxTime' => 60,
+                                    ],
+                                ],
+                            ],
+                            'timeLimits' => [
+                                'qti-type' => 'timeLimits',
+                                'maxTime' => 60,
+                            ],
+                        ],
+                    ],
                 ],
-                'qti_identifier' => 'test_id',
-                'test_qti_structure' => [
-                    'identifier' => 'test_id'
-                ]
             ],
-            $this->sut->normalize($test)->getBody()
-        );
+            'timeLimits' => [
+                'maxTime' => 3600,
+                'qti-type' => 'timeLimits',
+            ],
+        ];
+
+        return [
+            'Simple' => [
+                'testJson' => '{"identifier": "test_id"}',
+                'expectedNormalizedBody' => [
+                    'type' => ['document type'],
+                    'item_uris' => [
+                        'item://1',
+                        'item://2'
+                    ],
+                    'qti_identifier' => 'test_id',
+                    'test_qti_structure' => [
+                        'identifier' => 'test_id'
+                    ]
+                ],
+            ],
+            'Has categories and timers' => [
+                'testJson' => $qtiTestStructureJson,
+                'expectedNormalizedBody' => [
+                    'type' => ['document type'],
+                    'item_uris' => [
+                        'item://1',
+                        'item://2'
+                    ],
+                    'qti_identifier' => 'testIdentifier',
+                    'test_qti_structure' => $qtiTestStructureToBeIndexed
```

Comment:
It most certainly will increase the size. Hopefully under the 100mb limit https://www.elastic.co/guide/en/elasticsearch/reference/current/general-recommendations.html#maximum-document-size

Reply:
It will not reach 100mb for sure

Reply:
Actually I did not increase indexing data a lot, comparing with what has already being indexing only categories and timers were added